### PR TITLE
intbitset v2.2.1

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-=========
+===========
+ intbitset
+===========
+
+.. image:: https://img.shields.io/travis/inveniosoftware/intbitset.svg
+        :target: https://travis-ci.org/inveniosoftware/intbitset
+
+.. image:: https://img.shields.io/coveralls/inveniosoftware/intbitset.svg
+        :target: https://coveralls.io/r/inveniosoftware/intbitset
```
Comment (kaplun) [inline README.rst]: Note coverage does not work for cython modules

Comment (jirikuncar) [inline README.rst]: @kaplun it should be possible (see http://blog.behnel.de/posts/coverage-analysis-for-cython-modules.html)

Comment (kaplun) [inline README.rst]: Yep, but not out of the box, so the coverage report will not be useful (and probably greyedout)

Comment (kaplun) [inline README.rst]: Anyway would be indeed cool to implement it. I will have a look

**Comment by kaplun** [comment]
For me :shipit: . In this way we can release a new intbitset release and solve @dset0x troubles with segmentation fault in celery.

**Comment by jirikuncar** [comment]
@kaplun you are the only one with rights to PyPI. Hence it's up to you. Please merge always with `--ff-only`.

**Comment by kaplun** [comment]
Actually the releases on Pypi are now made directly through Travis so anybody having rights to push to intbitset on github can make a release.

**Comment by tiborsimko** [comment]
> Actually the releases on Pypi are now made directly through Travis

Having direct PyPI rights is useful regardless, e.g. what if the key that Travis uses gets broken into or gets revoked, e.g. what if one needs to invalidate released files or change PyPI settings, etc. In the case of other `inveniosoftware` packages, there are usually two or three people declared as owners for additional safety, e.g. in case people are away.

**Comment by kaplun** [comment]
@tiborsimko done.

**Pull request merged**